bootstrap/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM fedora:32
```

Comment:
nit: I would probably call this folder "dev" or something along those lines, in my mind bootstrap is more for, well, bootstrapping. This is just a container for development purposes AIUI, correct?